doc/rst/platforms/raspberrypi.rst

```diff
@@ -0,0 +1,142 @@
+
+In this example, we will use ``10.0.0.0`` as our subnet. If your regular network uses this, choose something different like ``192.168.10.0``.
+
+.. code-block:: bash
+
+    sudo nano /etc/dhcpcd.conf
+
+At the end of the file, add the lines:
+
+.. code-block:: bash
+
+    interface eth0
+    static ip_address=10.0.0.1/24
+
+For each node, replace the last digit of the subnet with a new unique value, such as 2, 3, and so on. You should be able to ping each node -- for example, from 10.0.0.1:
+
+.. code-block:: bash
+
+    ping 10.0.0.2
+
+1. Generate SSH keys
+
+On each node, generate an SSH key by running the command:
+
+.. code-block:: bash
+
+    ssh-keygen -t rsa
+
+Press RETURN for each prompt and do not create a passphrase when asked.
+
+4. Copy SSH keys to all nodes in the cluster
+
+For each non-master node, run:
+
+.. code-block:: bash
+
+    ssh-copy-id 10.0.0.1 
+
+to copy that node's SSH key to the master node. Once that has been done on all non-master nodes, copy the master node's SSH key to all other nodes in the cluster by running the same command from the master node, using the IP addresses of the other nodes as the command argument.
+
+5. Set up a shared filesystem
+
+In order for each node to reference the same Chapel executable, the executable must be stored in a shared filesystem that all of the nodes in the cluster have simultaneous access to. There are several ways to do this. 
+
+One way is to mount a flash drive and export it as a network file system to the other nodes in the cluster. The steps to achieve this are described in sections 4.1.1 to 4.1.2 from this `Medium article <https://glmdev.medium.com/building-a-raspberry-pi-cluster-784f0df9afbd>`_.
+
+--------------------------------------
+Configuring for Multi-Locale Execution
+--------------------------------------
+
+For more in-depth information about GASNet or multilocale execution with Chapel,
+refer to the `GASNet documentation`_ and :ref:`readme-multilocale` page,
+respectively.
+
+Ensure that the Chapel tree is installed in the shared filesystem (or that the Chapel tree is built on each node in your cluster), each node is running the same version of Chapel, and that all nodes in the cluster have SSH access to each other. 
+
+Complete these steps on the master node of your cluster.
+
+1. Set Chapel to use GASNet for multilocale execution
+   
+Set the following environment variable:
+
+.. code-block:: bash   
+
+    CHPL_COMM=gasnet
+
+2. Set up GASNet environment variables
+
+Set the following GASNet environment variables:
+
+.. code-block:: bash
+
+    GASNET_SPAWNFN=S # Job spawn mechanism, where 'S' means ssh/rsh-based spawn
+    GASNET_MASTERIP= # IP address of master node in your cluster, eg. 10.0.0.1
+    GASNET_SSH_SERVERS= # Space-delimited list of server names, eg. 10.0.0.1 10.0.0.2 10.0.0.3
+
+
+Notes
+-----
+
+The last version of the Chapel compiler recorded to successfully build on a Raspberry Pi is 1.28.0.
```

Comment:
@vlzambr2 - I believe this is more a limitation of the specific hardware used, not Chapel on Pis in general. For example, we have seen our latest main build on Pi hardware that includes >512MB of RAM.
 
Additionally, we generally avoid including version numbers in the docs like this because they tend to be forgotten and then not updated/revisited later. 

Would you mind either updating this PR to remove this note, or acknowledging that you are OK with us updating the docs post-merge by responding here?